backend/de.metas.manufacturing.rest-api/src/main/java/de/metas/handlingunits/rest_api/HandlingUnitsService.java

```diff
@@ -106,7 +112,8 @@ public JsonHU toJson(@NonNull final I_M_HU hu, @NonNull final String adLanguage)
 				.qrCode(toJsonHUQRCode(huId))
 				.jsonHUType(toJsonHUType(hu))
 				.products(getProductStorage(huContext, hu))
-				.attributes(getAttributes(huContext, hu))
+				.attributes2(jsonHUAttributes)
```

Comment:
maybe change the name of `attributes2` to `attributes` and change `attributes` to `attributesCodeAndValues`?
otherwise it's hard to memorize which is which..